machine/lenovo/lenovo_ne2572/machine.make

```diff
@@ -0,0 +1,51 @@
+# Lenovo NE2572
+
+ONIE_ARCH ?= x86_64
+SWITCH_ASIC_VENDOR = bcm
+
+VENDOR_REV ?= 0
+
+# Translate hardware revision to ONIE hardware revision
+ifeq ($(VENDOR_REV),0)
+  MACHINE_REV = 0
+else
+  $(warning Unknown VENDOR_REV '$(VENDOR_REV)' for MACHINE '$(MACHINE)')
+  $(error Unknown VENDOR_REV)
+endif
+
+# The VENDOR_VERSION string is appended to the overal ONIE version
+# string.  HW vendors can use this to appended their own versioning
+# information to the base ONIE version string.
+VENDOR_VERSION = .0.3
+
+# Vendor ID -- IANA Private Enterprise Number:
+# http://www.iana.org/assignments/enterprise-numbers
+# Lenovo 
+VENDOR_ID = 19046
+
+# Enable the i2ctools and the onie-syseeprom command for this platform
+I2CTOOLS_ENABLE = yes
+I2CTOOLS_SYSEEPROM = no
```

Comment:
`I2CTOOLS_SYSEEPROM = no` is the default already.

Funny, with this set to `no` the i2ctools patches above are not used at all.

Also you have defined the busybox implementation of onie-syseeprom.

I think the thing to do here is completely remove the i2ctools directory from this patch.

Reply:
Same comments for the previous `lenovo_ne10032` platform.  Looks like you can get rid of the `i2ctools` subdirectory.  It is not being used.

Reply:
I will comment out I2CTOOLS_SYSEEPROM = no. thank you. I will also delete i2ctools directory